truffle-config.js

```diff
@@ -39,9 +39,9 @@ module.exports = {
   ...truffleConfig({
     mnemonic,
     privateKey,
-    urlRinkeby: "https://rinkeby.infura.io/v3/".concat(infuraKey),
+    urlRinkeby: "https://node.rinkeby.gnosisdev.com",
```

Comment:
I'm not sure we should advertise our full nodes in a public repo. This could easily lead to more traffic than we want/expect.

Reply:
Then we should also do have it dex-services :)

But I agree and remove it over here